turbinia/client.py

```diff
@@ -251,47 +305,61 @@ def format_task_status(
       user (string): The user of the request we want tasks for.
       all_fields (bool): Include all fields for the task, including task,
           request ids and saved file paths.
+      full_report (bool): Generate a full markdown report instead of just a
```

Comment:
markdown_report instead? Maybe there will be other formats of the report in the future?

Reply:
I guess the help text for this flag is a little misleading, so I'll fix that, but this doesn't actually swap it from a normal report to markdown report (all report data will be in markdown either way), it just adds extra detail to make it a "full" report instead of a summary.  I guess I could change it to --detailed_report or something like that.